views/js/qtiCreator/helper/itemLoader.js

```diff
@@ -86,6 +91,11 @@ define([
                         //tag the item as a new one
                         newItem.data('new', true);
 
+                        //add languages list to the item
+                        if (data.languagesList) {
+                            newItem.attr('languagesList', data.languagesList);
```

Comment:
same as above, please use `newItem .data()` instead.

Reply:
Same as above. I'll keep you updated.